src/main/java/fi/okm/jod/yksilo/config/OpenApiConfig.java

```diff
@@ -30,6 +31,19 @@
 @Slf4j
 class OpenApiConfig {
 
+  @Bean
+  public GroupedOpenApi yksiloApi() {
+    return GroupedOpenApi.builder().group("JOD Yksilo API").pathsToMatch("/api/**").build();
```

Comment:
the group name is part of the url, so it should be "url compatible" for easier access (e.g. just yksilo)

apparently the api customizer is not applied automatically, should apply it manually here (addOpenApiCustomizer)